src/utils/generateUniqueToken.ts

```diff
@@ -0,0 +1,14 @@
+export const generateUniqueToken = async () => {
+  const uuidToken = crypto.randomUUID();
+  const randomNumber = Math.floor(Math.random() * 1000000);
```

Comment:
Can we please store large numbers in a constant?

Reply:
isn't it already a constant

Reply:
I am talking about the `1000000`

Reply:
The application did not respond only comes when the application is just now deployed. if you try on my bot now it won't show app did not respond

also for the package lock, I installed some packages and then removed them as they were not required. If you try to see the changes you can see there is a change in integrity as a package is upgraded. That will cause package-lock.json to change. it did not change package.json as the package updated was a dependency/funding for a working package.

As for the constant, just for 1 number, and that too is used in a single place I don't think it requires a separate constant and import.

Reply:
Do we also verify the generated token at any place? 
If so, how will we do it as the encoded string (at line 6) has a random value i.e `randomNumber?

Reply:
That's a great question

Reply:
hi @yesyash We can not verify something completely random. To test it out we can store the random token and then check if the new token generated is different.

Reply:
That does not answer my first question are we verifying the generated token at any place ?

Reply:
yes it will be verified in my site